packages/components/radio/src/radio-button.vue

```diff
@@ -2,7 +2,7 @@
   <label
     :class="[
       ns.b('button'),
-      ns.is('active', modelValue === label),
+      ns.is('active', modelValue === getValueOrLabel(props)),
```

Comment:
**Suggestion:** can we make this a computed value?

Something like the following.

```ts
const actualValue = computed(() => getValueOrLable(props));
```

Reply:
Yes, I tried to make a computed value in useRadio, but some files don't use useRadio. Or should I make `actualValue` in each files?